plugins/modules/pnp_intent.py

```diff
@@ -934,7 +980,7 @@ def get_diff_deleted(self):
                 function='get_device_list',
                 params={"serial_number": device["deviceInfo"]["serialNumber"]}
             )
-
+            self.log("Device details for the serial number {0} are {1}".format(device["deviceInfo"]["serialNumber"], str(multi_device_response)), "DEBUG")
```

Comment:
self.log("Response from 'claim_a_device_to_a_site' API for claiming: {0}".format(str(claim_response)), "DEBUG")

Reply:
Resolved